scripts/convertSituation/convertTo2Tonnes.ts

```diff
@@ -0,0 +1,85 @@
+import NGCRules from '@incubateur-ademe/nosgestesclimat/public/co2-model.FR-lang.fr.json'
+import type { NodeValue } from '@incubateur-ademe/nosgestesclimat'
+import path from 'path'
+import Engine from 'publicodes'
+import { readYAML } from '../utils'
+import type { Situation } from '../../src/types/types'
+
+type Props = {
+  situation: Situation
+}
+
+export default function convertTo2Tonnes({ situation }: Props) {
```

Comment:
Could you make it more generic ?

Reply:
Not really because output for `NGCSituationAs2T` is very specific. I think it's easier to create (and test) one conversion script for one partnership